src/clj/rems/css/styles.clj

```diff
@@ -653,8 +658,8 @@
    [:.form-group {:text-align "initial"}
     ;; make fieldset legends look the same as normal labels
     [:legend {:font-size "inherit"}]]
-   [:#application-fields
-    [:.application-field-label {:font-weight "bold"}]]
+   [:label.application-field-label {:font-weight "bold"}]
+   [:div.field-info {:font-weight "400"}]
```

Comment:
Above you have `[:.field-info {:white-space :pre-wrap}]` that you can combine here with `[:div.field-info {:font-weight "400"}]`.

Reply:
this seems done now